Application/mobile/app_code/lib/pages/map/map_state.dart

```diff
@@ -1,62 +1,293 @@
+import 'dart:convert';
+
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:geolocator/geolocator.dart';
 import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'package:http/http.dart' as http;
 import 'package:permission_handler/permission_handler.dart';
+import 'package:provider/provider.dart';
+import 'package:risu/components/loader.dart';
+import 'package:risu/components/showModalBottomSheet.dart';
+import 'package:risu/globals.dart';
+import 'package:risu/pages/article/details_page.dart';
+import 'package:risu/pages/article/list_page.dart';
+import 'package:risu/pages/container/container_list.dart';
 import 'package:risu/utils/errors.dart';
+import 'package:risu/utils/providers/theme.dart';
 
 import 'map_page.dart';
 
 class MapPageState extends State<MapPage> {
   GoogleMapController? mapController;
-  final bool displayGoogleMap = true;
+  PermissionStatus? permission;
+  List<ContainerList> containers = [];
+  final LoaderManager _loaderManager = LoaderManager();
+  List<dynamic> listItems = [];
+  Map<String, dynamic> containersData = {};
+  Color dividerColor = Colors.black12;
 
-  LatLng _center = const LatLng(37.7749, -122.4194);
+  LatLng _center = const LatLng(33.139469, -117.161148);
```

Comment:
What is it for ?

Reply:
It was just for me, so it was easier to go to my location. I fixed it by setting the value at the location of Epitech Nantes